src/platform/forms-system/test/js/components/Telephone.unit.spec.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import Telephone, { CONTACTS } from '../../../src/js/components/Telephone';
+
+describe('Widget <Telephone />', () => {
+  // custom numbers
+  it('should render a number', () => {
+    const wrapper = shallow(<Telephone contact="8005551212" />);
+    const props = wrapper.props();
+    expect(props.href).to.equal('tel:+18005551212');
+    expect(props['aria-label']).to.equal('800. 5 5 5. 1 2 1 2');
+    expect(wrapper.text()).to.equal('800-555-1212');
+    wrapper.unmount();
+  });
+  it('should render a number with a leading "1"', () => {
+    const wrapper = shallow(<Telephone contact="1-800-555-1000" />);
+    const props = wrapper.props();
+    expect(props.href).to.equal('tel:+18005551000');
+    expect(props['aria-label']).to.equal('800. 5 5 5. 1000');
+    expect(wrapper.text()).to.equal('800-555-1000');
+    wrapper.unmount();
+  });
+  it('should throw an error when not passed a number', () => {
+    expect(() => {
+      const wrapper = shallow(<Telephone />);
+      wrapper.unmount();
+    }).to.throw('Telephone: "" does not match the pattern (###-###-####)');
```

Comment:
The error would be more helpful if explained which prop was missing

Reply:
That error is actually showing the passing in number & pattern, but since there was no number passed in that test, it shows an empty string